### PR TITLE
MODFEE-53] Publish fee/fine balance change event

### PR DESCRIPTION
**Pull request by OleksandrVidinieiev**
**Purpose**
Introduce integration with mod-pubsub. Publish events when account is created, changed or deleted.

**Resolves**
[MODFEE-53](https://issues.folio.org/browse/MODFEE-53)

### CODE REVIEW

File: src/test/java/org/folio/rest/impl/APITests.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+
+public abstract class APITests {
```
Comment (roman-barannyk) [inline src/test/java/org/folio/rest/impl/APITests.java:34]: It seems like this class doesn't have any abstract methods. Why is it abstract?

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -620,7 +620,12 @@
...
         {
           "methods": ["POST"],
-          "pathPattern": "/_/tenant"
+          "pathPattern": "/_/tenant",
+          "modulePermissions": [
+            "pubsub.event-types.post",
+            "pubsub.publishers.post",
+            "pubsub.subscribers.post"
+          ]
         }, {
```
Comment (alexanderkurash) [inline descriptors/ModuleDescriptor-template.json:629]: Please add PubSub interfaces dependencies and permissions

File: src/test/java/org/folio/rest/impl/APITests.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  protected static final String OKAPI_TENANT = "test_tenant";
+  protected static final String OKAPI_TOKEN = buildToken();
+  protected static final String OKAPI_HEADER_URL = "x-okapi-url";
+
+  protected static Vertx vertx;
+  protected static OkapiClient okapiClient;
+  protected static TenantClient tenantClient;
+
+  @Rule
+  public WireMockRule wireMock = new WireMockRule(
```
Comment (alexanderkurash) [inline src/test/java/org/folio/rest/impl/APITests.java:51]: Move wireMock to the class where it's being used

File: descriptors/ModuleDescriptor-template.json (hunk 2 above)
Comment (OleksandrVidinieiev) [reply descriptors/ModuleDescriptor-template.json:648]: added

File: src/test/java/org/folio/rest/impl/APITests.java (hunk 3 above)
Comment (OleksandrVidinieiev) [reply src/test/java/org/folio/rest/impl/APITests.java]: done

**Pull request merged**